scss/components/_ad-screen.scss

```diff
@@ -1,6 +1,5 @@
 .adScreen {
   cursor: default;
-  z-index: 13000;
```

Comment:
Was this intentional? Can we do git blame and make sure we are not deleting other issue's fix?